airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/UploadWorkers.java

```diff
@@ -47,15 +47,19 @@ public void start() {
   }
 
   private void retrieveWork() {
+    // todo (cgardens) - i'm not convinced this makes sense. as we get close to the limit, we should
+    // flush more eagerly, but "flush all" is never a particularly useful thing in this world.
```

Comment:
agree. i think the hard req for flushAll is when we are closing down the destination.